recipes/deepforeset/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "DeepForest" %}
+{% set version = "0.2.17" %}
+{% set sha256 = "b03dad820445814df03f67960b997a125e2a6861be74c7221565149cebcd11f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py==37]
```

Comment:
So this works on Python 3.8?

Reply:
For python 3.5 and above. For some reason it was failing to work on p3.7 locally

Reply:
Does ot fail on CI on 3.7?